solutions/go/ping-pong/diff/app/server.go.diff

```diff
@@ -0,0 +1,30 @@
+@@ -1,20 +1,28 @@
+ package main
+
+ import (
+ 	"fmt"
+ 	"net"
+ 	"os"
+ )
+
+ func main() {
+ 	l, err := net.Listen("tcp", "0.0.0.0:6379")
+ 	if err != nil {
+ 		fmt.Println("Failed to bind to port 6379")
+ 		os.Exit(1)
+ 	}
+-	_, err = l.Accept()
++
++	conn, err := l.Accept()
```

Comment:
missing `defer conn.Close()` after this to make it "standard"

Reply:
Done 👍 https://github.com/codecrafters-io/build-your-own-redis/pull/34.